cassandra/cluster.py

```diff
@@ -938,6 +939,17 @@ def default_retry_policy(self, policy):
     establish connection pools. This can cause a rush of connections and queries if not mitigated with this factor.
     """
 
+    experimental_tablet_refresh_time = 60
```

Comment:
wouldn't it be enough to put the a disclaimer on the docstring, that it's experimental.

and not in the parameter name ? even when it will go out of experimental will need to leave that name for backward compatibility for quite a while... (and it would force to change across the board where we gonna test this)

Reply:
I don't think it matters - I requested name changes to `experimental_` and the comments when I thought that we are going to merge it. It is my understanding that we won't, because Tablets design (specifically, Tablets interactions with drivers) is not finished and will change (e.g. we probably won't query `system.tablets` table) - so we'll adjust the PRs according to Tablets changes in Scylla and merge when ready.

Reply:
@Lorak-mmk is right, we want to get rid of periodic check before merging.